hail/src/main/scala/is/hail/io/bgen/LoadBgen.scala

```diff
@@ -47,10 +46,10 @@ case class BgenFileMetadata(
 )
 
 object LoadBgen {
-  def readSamples(hConf: org.apache.hadoop.conf.Configuration, file: String): Array[String] = {
-    val bState = readState(hConf, file)
+  def readSamples(fs: FS, file: String): Array[String] = {
+    val bState = readState(fs, file)
     if (bState.hasIds) {
-      hConf.readFile(file) { is =>
+      fs.readFile(file) { is =>
         val reader = new HadoopFSDataBinaryReader(is)
```

Comment:
Is this OK? Will this work on other non-hadoop input streams?

Reply:
Yes, readFile returns a Java InputStream